Client/Application/AppDelegate.swift

```diff
@@ -131,6 +133,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         adjustIntegration = AdjustIntegration(profile: profile)
 
+        // We need to check if the app is a clean install to use for
+        // preventing the What's New URL from appearing.
+        if getProfile(application).prefs.stringForKey(LatestAppVersionProfileKey) == nil {
```

Comment:
This is not correct. If `LatestAppVersionProfileKey` is not set, which it is not when you upgrade from `1.4`, we set it to `2.0`, which means that we never detect the upgrade. Because both `LatestAppVersionProfileKey` and `AppInfo.appVersion` will be `2.0` now.